doc/man/8/rados.rst

```diff
@@ -69,18 +85,154 @@ Options
 
    Uses the striping API of rados rather than the default one.
    Available for stat, stat2, get, put, append, truncate, rm, ls
-   and all xattr related operation
+   and all xattr related operation.
+
+.. option:: -o object_size
+
+   Set the object size for put/get ops and for write benchmarking.
 
-.. option:: -O object_size
+.. option:: --max-objects
 
-   Set the object size for put/get ops and for write benchmarking
+   Set the max number of objects for write benchmarking.
 
 .. option:: --lock-cookie locker-cookie
 
    Will set the lock cookie for acquiring advisory lock (lock get command).
    If the cookie is not empty, this option must be passed to lock break command
    to find the correct lock when releasing lock.
 
+.. option:: --target-locator
+
+   Use with cp to specify the locator of the new object.
+
+   .. option:: --target-nspace
+
+   Use with cp to specify the namespace of the new object.
+
+
+Bench options
+=============
+
+   .. option:: -t N, --concurrent-ios=N
```

Comment:
why do you start to indent the options here?

Reply:
My mistake. Commiting the fix.